client/src/ui/search-input.js

```diff
@@ -137,13 +139,18 @@ function searchInput(parentNode, options) {
                 "<a href='https://galaxyproject.org/tutorials/histories/#advanced-searching' target='_blank'>",
                 _l("the Hub"),
                 '.</a></p>" title="',
-                _l("search tips"),
-                '"></span>',
+                _l("Search tips"),
+                '"></a>',
             ].join("")
         )
-            .tooltip({ placement: "bottom" })
+            .tooltip({ placement: "top", trigger: "hover" })
             .click(function () {
-                $('[data-toggle="advSearchPopover"]').popover({ html: true, container: ".history-right-panel" });
+                $('[data-toggle="advSearchPopover"]').popover({
+                    trigger: "focus",
+                    html: true,
+                    container: ".history-right-panel",
+                });
+                $('[data-toggle="advSearchPopover"]').popover("show");
```

Comment:
Can this be chained so we don't need to select twice?

Reply:
Not that I was finding in any of the documentation, and I tried for a while to combine the commands. Might have just missed something, though...